themes-default/slim/src/store/modules/shows.js

```diff
@@ -84,17 +85,40 @@ const actions = {
     getShows(context, shows) {
         const { commit, dispatch } = context;
 
-        // If no shows are provided get the first 10k
+        // If no shows are provided get the first 1000
         if (!shows) {
-            const params = {
-                limit: 10000
-            };
-            return api.get('/series', { params }).then(res => {
-                const shows = res.data;
-                return shows.forEach(show => {
+            return (async() => {
+                const limit = 1000;
+                const page = 1;
+                const params = {
+                    limit,
+                    page
+                };
+
+                // Get first page
+                try {
+                    const response = await api.get('/series', { params });
```

Comment:
Unlike Python, `response` will only be available within the `try { }` block.
If you need to access response outside of that block, you need to define `let response = null;` before the block, and assign the `await`-ed value inside the block.